unity/Assets/Scripts/Quest/Quest.cs

```diff
@@ -1314,6 +1314,7 @@ public Tile(QuestData.Tile questTile, Game gameObject) : base(gameObject)
             // We need this for the 3x2 MoM tiles because they don't have square pixels!!
             if (mTile.aspect != 0)
             {
+                vPPS = Mathf.Min(newTex.width, (float)newTex.height) / 7f;
```

Comment:
I tested Android Windows for errors, but might want to check it out, too. Maybe there are sideeffects I don't know of.

Reply:
dds = 2048x1024
pvr = 1024x512
 
1024 / 7 = 146.28571428571428571428571428571 = 146.28571 (cutted)
512 / 7 = 73.142857142857142857142857142857 = 73.14285 (cutted)

ppi = Math.min(img.height, img.width) / 7

Reply:
While this would work, I don't like it because we are assuming that all tiles that have a stange aspect are these specific tiles.  Right now this happens to be true, but I think it is a poor solution.

 For an easyish solution I would go with ```pps=*0.5``` for these tiles, detect the '*' and use the number as a scale from the default.

OR

use image=tile.dds, image2=tile.pvr with pps=146, pps2=73

I you like I can do this.

Reply:
The second option does not work for MoM01 tiles as they are jpg, so the selection doesn't work.

Reply:
The first option would be great, but I have no idea how to make it work.

Reply:
I probably didn't explain it well.

<pre>
image=[file]
image2=[anotherFile]
pps=[number]
pps2=[anotherNumber]
</pre>

If image doesn't match we use image2 (like currently), in this case we read pps2 instead of pps.  This will allow us to specify different pps for .dds and .pvr files (but the actual extension is not considered).

For tiles included with Valkyrie (jpg) this could be used, but isn't needed because the pps will always be the same.  This could also be used for all imported tile images instead of the change to default when running on android, optionally.

Reply:
I will do something, hopefully tonight (maybe 10:00 UTC).

Reply:
Don't worry, will try on. Got a good result using the Foyer tile. Reverted and changed:

   ```
     if (content.ContainsKey("pps"))
        {
            if (content["pps"].StartsWith("*"))
            {
                float.TryParse(content["pps"].Remove(0, 1), out pxPerSquare);
                pxPerSquare *= Game.Get().gameType.TilePixelPerSquare();
            }
            else
            {
                float.TryParse(content["pps"], out pxPerSquare);
            }
        }
        else
        {
            pxPerSquare = Game.Get().gameType.TilePixelPerSquare();
        }
```

However, I will have to check for side effects on the other tiles.

Reply:
Looks good, thanks.